site/src/pages/RoadmapPage/sidebar/ProgramRequirementsList.tsx

```diff
@@ -183,7 +206,13 @@ const GroupRequirement: FC<GroupRequirementProps> = ({ data, takenCourseIDs }) =
       )}
       {open &&
         data.requirements.map((r, i) => (
-          <ProgramRequirementDisplay key={i} requirement={r} nested takenCourseIDs={takenCourseIDs} />
+          <ProgramRequirementDisplay
+            key={i}
+            storeKey={storeKey + '-' + i}
```

Comment:
minor nit – maybe use a template string?
```suggestion
            storeKey={`${storeKey}-${i}`}
```